lib/slack/handlers.ex

```diff
@@ -32,7 +32,7 @@ defmodule Slack.Handlers do
   end
 
   def handle_slack(%{type: "channel_left", channel: channel}, slack) do
```

Comment:
Can we rename `channel` to `channel_id` to make it clear what slack is giving us?